src/downloader.ts

```diff
@@ -42,6 +41,7 @@ const downloadBiomeVersion = async (
 ): Promise<Uri | undefined> => {
 	debug("Downloading Biome version", { version });
 
+	const { default: ky } = await import("ky");
```

Comment:
This is to fix the `ky` issue.

Reply:
Weird, never had any issue, sounds good.